### ISSUE TITLE
installation on proxmox / debian bullseye fails

### ISSUE THREAD

**Issue by matti** [k0sproject/k0s#1248]
latest proxmox installed (based on debian bullseye, with maybe different kernel?)

install from docs:

```
curl -sSLf https://get.k0s.sh | sudo sh
sudo k0s install controller --single
sudo k0s start
```

results in Node NotReady and `dmesg` says:

```
[ 4491.034087] overlayfs: upper fs does not support RENAME_WHITEOUT.
[ 4491.034179] overlayfs: upper fs missing required features.
[ 4498.037931] overlayfs: upper fs does not support RENAME_WHITEOUT.
[ 4498.038019] overlayfs: upper fs missing required features.
[ 4503.177809] overlayfs: upper fs does not support RENAME_WHITEOUT.
```

proposal: detect required features on install

**Comment by jnummelin** [comment]
We definitely should add the system check (`k0s sysinfo`) as a pre-flight check on install.

**Comment by soider** [comment]
Closed in favor #1289
